pkg/lib/operation/project/remote/table/import/operation.go

```diff
@@ -23,31 +25,55 @@ type Options struct {
 	Columns         []string
 	IncrementalLoad bool
 	WithoutHeaders  bool
+	PrimaryKey      []string
 }
 
 func Run(ctx context.Context, o Options, d dependencies) (err error) {
 	ctx, span := d.Tracer().Start(ctx, "kac.lib.operation.project.remote.table.import")
 	defer telemetry.EndSpan(span, &err)
 
-	job, err := d.KeboolaProjectAPI().LoadDataFromFileRequest(o.TableID, o.FileID, getOptions(&o)...).Send(ctx)
-	if err != nil {
-		return err
-	}
+	if !checkTableExists(ctx, d, o.TableID) {
```

Comment:
Table import vytvoří bucket a table (přímo pomocí `from file` API callu) pokud neexistuje. Používám rollback aby se ten bucket smazal, pokud to failne dál.